rpc/wrtc_client_stream.go

```diff
@@ -194,8 +208,18 @@ func init() {
 }
 
 func (s *webrtcClientStream) writeMessage(m interface{}, eos bool) (err error) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	if s.sendClosed {
+		return io.ErrClosedPipe
+	}
+	if eos == true {
+		s.sendClosed = true
```

Comment:
both reset and this cause send closures preventing some double write errors on a closed stream